selfdrive/car/ford/radar_interface.py

```diff
@@ -1,72 +1,96 @@
 #!/usr/bin/env python3
+from math import sin
 from cereal import car
-from common.conversions import Conversions as CV
 from opendbc.can.parser import CANParser
 from selfdrive.car.ford.values import CANBUS, DBC
 from selfdrive.car.interfaces import RadarInterfaceBase
 
-RADAR_MSGS = list(range(0x500, 0x540))
+RADAR_MSGS = list(range(0x120, 0x15F))  # 64 points
```

Comment:
oops, maybe the range should be `range(0x120, 0x15F + 1)` to include the last message...